src/extension.ts

```diff
@@ -301,32 +291,55 @@ function configureRunTestFromCodeLens() {
   vscode.commands.registerCommand(Commands.RUN_TEST_FROM_CODELENS, runFromCodeLens);
 }
 
-export function activate(context: ExtensionContext): void {
-  testElixir();
-  detectConflictingExtension("mjmcloug.vscode-elixir");
-  // https://github.com/elixir-lsp/vscode-elixir-ls/issues/34
-  detectConflictingExtension("sammkj.vscode-elixir-formatter");
-
-  configureRunTestFromCodeLens()
-  configureCopyDebugInfo(context);
-  configureExpandMacro(context);
-  configureDebugger(context);
-  configureTerminalLinkProvider(context);
-
+function startClient(context: ExtensionContext, clientOptions: LanguageClientOptions): LanguageClient {
   const command =
     os.platform() == "win32" ? "language_server.bat" : "language_server.sh";
 
   const serverOpts = {
     command: context.asAbsolutePath("./elixir-ls-release/" + command),
   };
 
-  // If the extension is launched in debug mode then the debug server options are used
-  // Otherwise the run options are used
   const serverOptions: ServerOptions = {
     run: serverOpts,
     debug: serverOpts,
   };
 
+  let displayName;
+  if (clientOptions.workspaceFolder) {
+    console.log(`starting for ${clientOptions.workspaceFolder!.name} with options`, clientOptions)
```

Comment:
Did you mean to leave these console.log statements in here?

Reply:
I guess they can stay for debug purposes. They may need some polishing though

Reply:
k, that sounds good